packages/block-library/src/navigation-menu/theme.scss

```diff
@@ -3,4 +3,31 @@
 	ul li {
 		list-style: none;
 	}
+
+	& > ul {
+		// Sub-menus Flyout
+		& > li > ul {
+			background: #000;
```

Comment:
Whether this is black or white is more or less a matter of "light or dark mode", I don't personally have a strong preference there, but it highlights a problem with both approaches. In light mode, the link colors will only work when they are dark. In dark mode, the links will only work when light. Aside from any other visuals, this feels like a problem worth tackling.

Is there a way we can style the flyout menu so that it works with either? I don't know if `background: inherit;` will work here, I don't think it will — but it sure would be nice if it could set the background to something theme-inherited, without us jumping through hoops to make it happen.

Absent that, I wonder if we should just lean in to the fact that we are styling the hover item, and apply _both_ a background color, and a text color?

I would always lean towards the former, but that seems to be the biggest challenge to fix.